### ISSUE TITLE
Does not show up on 10.12.3 (16D32)

### ISSUE THREAD

**Issue by wingsstudio** [bikkelbroeders/TouchBarDemoApp#93]
Is that mean 10.12.3(16D32) cannot be support?

**Comment by robbertkl** [comment]
Everything after 10.12.1 should work. In fact, I just tested it on 10.12.3 and it worked fine.

**Comment by wingsstudio** [comment]
Is there any other thing to set to make app work didn't show in ReadMe.md ?

**Comment by robbertkl** [comment]
No, just install it and you should be good to go. Did you check the menubar icon to see what toggle key is set? Try a different key.

**Comment by wingsstudio** [comment]
In fact, before submit issues I have try every key . But no one was available .
There is a strange thing . On the first time open the app , it didn't show up a window for requiring me to  give accessibility permissions , Is that the issues's reason?
But the app didn't work even I add the app to accessibility list ...

**Comment by robbertkl** [comment]
What version of TouchBarDemoApp do you have?

**Comment by wingsstudio** [comment]
V16

**Comment by robbertkl** [comment]
Every version since v1.2 does not ask about accessibility anymore! So I'm not really sure how it could have asked on v1.6.

**Comment by wingsstudio** [comment]
It's so sad that the app cannot run on my macbook and even now I haven't found out the reason.
:-(

**Comment by robbertkl** [comment]
Are you running any other accessibility tools, like for example BetterTouchTool?

**Comment by wingsstudio** [comment]
In fact, I run a lot of accessibility tools...
I will try to run the app without open any other app if I have time.
After all , thanks for talking .
:-)

**Comment by Klawens** [comment]
It's not working on 10.12.3 (16D32)

**Comment by midi1996** [comment]
Working fine here with 10.12.4 (16E195). Using v1.6. As said before, disable other Accessibility apps and try again, and change the shortcut to anything other than Fn and try again.
